active/online-community.md

```diff
@@ -0,0 +1,97 @@
+# Online Community Working Group
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The [Django Forum](https://forum.djangoproject.com/) and the [Django Discord](https://discord.gg/xcRH6mN4fa) are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others.
+We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+  - ensure consistency that our online Django communities are welcoming and safe spaces
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms. Examples of these processes include
+    - Proposals for adding/removing moderators & other roles and the adding/removing of roles directly
+    - Training for moderators & other roles as required
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+
+- Which actions will the WG take back to the Board for votes?
+  - Proposal to trial a new platform
+  - Proposal to migrate to a new platform
+  - Proposal to retire an existing platform that is in use
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- Administrator Privileges on approved community platforms with the following conditions to create redundancy and protect the community from any single person holding too much power:
```

Comment:
If I read these two lines together it kinda sounds a bit like the group wants to have admin privileges across all platforms rather than expressing restrictions on people (in the group or not). I don't think this is intended and should probably be clarified.

Reply:
I have altered the wording of the first sentence, but agree it's not the most clear. My intention was to ensure that there is someone with admin privileges with in the group and that it's not all with a single member.